### PR TITLE
Package apron.v0.9.12

### PR DESCRIPTION
**Pull request by antoinemine**
### `apron.v0.9.12`
APRON numerical abstract domain library
Apron is a library to represent properties of numeric variables, such as variable bounds or linear relations between variables, and to manipulate these properties through semantic operations, such as variable assignments, tests, conjunctions, entailment. Apron is intended to be used in static program analyzers, in order to infer invariants of numeric variables, i.e., properties that hold for all executions of a program. It is based on the theory of Abstract Interpretation.

---
* Homepage: https://antoinemine.github.io/Apron/doc/
* Source repo: git+https://github.com/antoinemine/apron.git
* Bug tracker: https://github.com/antoinemine/apron/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

File: packages/apron/apron.v0.9.12/opam [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  ]
+  [make "-j%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+remove: [
+  ["ocamlfind" "remove" "apron"]
+  ["rm" "-r" "-f" "%{share}%/apron"]
+]
```
Comment (kit-ty-kate) [inline packages/apron/apron.v0.9.12/opam]: The `remove` field is unnecessary since opam 2.0

File: packages/apron/apron.v0.9.12/opam [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  "ocamlfind"
+  "camlidl"
+  "mlgmpidl"
+  "ocamlbuild" {build}
+  "conf-perl"
+]
+depopts: [
+  "conf-ppl"
+]
+flags: light-uninstall
```
Comment (kit-ty-kate) [inline packages/apron/apron.v0.9.12/opam]:
```suggestion
```
This is unnecessary once the `remove` field has been removed.

File: packages/apron/apron.v0.9.12/opam [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+install: [
+  [make "install"]
+]
+remove: [
+  ["ocamlfind" "remove" "apron"]
+  ["rm" "-r" "-f" "%{share}%/apron"]
+]
+depends: [
+  "ocaml"
+  "ocamlfind"
```
Comment (kit-ty-kate) [inline packages/apron/apron.v0.9.12/opam]:
```suggestion
  "ocamlfind" {build}
```
It seems like this package only uses `ocamlfind` as binary during compilation. Therefore it should be safe to tag is as a build dependency

**Comment by kit-ty-kate** [comment]
Note to self: The `v` prefix is necessary since previous versions are of the form `YYYYMMDD`

**Comment by antoinemine** [comment]
I've applied the suggested changes, thanks! I also fixed a java-11 related installation error. Tell me if you need other fixes upstream.

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**